code/__DEFINES/atmospherics.dm

```diff
@@ -40,8 +40,8 @@
 /// liters in a cell
 #define CELL_VOLUME				2500
 
-/// liters in a normal breath
-#define BREATH_VOLUME			0.5
+/// liters in a normal breath. note that breaths are taken every 8 seconds
```

Comment:
After our discussions on this, I feel we might as well expand to explain why it's 8 seconds. That fact seemed to draw some confusion even from other coders.

```suggestion
/// liters in a normal breath. note that breaths are taken once every 4 life ticks, which is 8 seconds
```